### PR TITLE
Adsk Contrib - Default to C++14 and remove OIIO from ocioperf

### PR DESCRIPTION
**Pull request by hodoulp**
Signed-off-by: Patrick Hodoul <Patrick.Hodoul@autodesk.com>

The pull request moves the default C++ version from 11 to 14 as discussed (but C++11 version remains supported) and removes the OpenImageIO dependency from the `ocioperf` command line tool (refer to #1314).

The `ocioperf` changes/improvements are:
- it uses a 4K image from [-1.0f, 2.0f] in float case.
- user can disable optimizations (default is false)
- user can specify the input and output bit-depths (defaults are float)

### CODE REVIEW

**Review by michdolan — APPROVED** [review]

**Comment by hodoulp** [comment]
Following discussions between @doug-walker and me we would like to improve the image generation (i.e. the slow gradient proposal does not capture well typical images). Stay tuned.

**Comment by hodoulp** [comment]
Sorry for the delay but we now have a better image generation.

**Comment by hodoulp** [comment]
@remia @michdolan There are still four Linux CI builds pending. Any idea? Can I still merge the pull request?

File: src/apps/ocioperf/main.cpp [hunk 1]
```diff
@@ -573,101 +459,197 @@ int main(int argc, const char **argv)
...
+        std::vector<float> img_f32_ref;
+        std::vector<uint16_t> img_ui16_ref;
+
+        // Generate a synthetic image by emulating a LUT3D identity algorithm that steps through
+        // many different colors.  Need to avoid a constant image, simple gradients, or anything
+        // that would result in more cache hits than a typical image.  Also, want to step through a 
+        // wide range of colors, including outside [0,1], in case some algorithms are faster or
+        // slower for certain colors.
+
+        static constexpr size_t length   = 64;
```
Comment (remia) [inline src/apps/ocioperf/main.cpp:487]: [Question] How was this size picked? 64^3 is way lower than 3840x2160 so I think you have lot of repetition in the image but that might be by design?

**Comment by remia** [comment]
> @remia @michdolan There are still four Linux CI builds pending. Any idea? Can I still merge the pull request?

I'm fine with merging on my side, it seems like a GitHub Action internal error.

File: src/apps/ocioperf/main.cpp (hunk 1 above)
Comment (doug-walker) [reply src/apps/ocioperf/main.cpp:487]: Good point Remi, I would suggest increasing it to 201.

**Review by doug-walker — APPROVED** [review]

**Pull request merged**